rskj-core/src/main/java/co/rsk/core/bc/PendingStateImpl.java

```diff
@@ -44,15 +45,15 @@
 import java.util.concurrent.ScheduledFuture;
 import java.util.concurrent.TimeUnit;
 
-import static org.ethereum.crypto.HashUtil.sha3;
+import static org.ethereum.crypto.HashUtil.keccak256;
```

Comment:
Please fix unused imports!
You also replace some static usages with fully qualified usages later on, maybe you want to make that coherent.

Reply:
either all `HashUtil.keccak256` or all `keccak256`, I'm fine with both.